### PR TITLE
Rename "People" to "Members" in organization page and use a better icon

### LINKED ISSUE #22931: The link of org people/member page
### Feature Description

Org people page's link is `domain/org/{OrgName}/members`.
Maybe `domain/org/{OrgName}/people` is better or just change the tab name from `People` to `Members`?

### Screenshots

_No response_

### PR DESCRIPTION
**Pull request by yardenshoham**
`member` is how it's named in the code

Closes #22931 

Before | After
--- | ---
image | image

### CODE REVIEW

**Comment by yardenshoham** [comment]
Should I delete all `org.people` from the `options/locale` files? Or maybe replace `people` -> `members`?

File: templates/org/menu.tmpl [hunk 1]
```diff
@@ -18,7 +18,7 @@
 		{{end}}
 		{{if .IsOrganizationMember}}
 			<a class="{{if $.PageIsOrgMembers}}active {{end}}item" href="{{$.OrgLink}}/members">
-				{{svg "octicon-organization"}}&nbsp;{{$.locale.Tr "org.people"}}
+				{{svg "octicon-person"}}&nbsp;{{$.locale.Tr "org.members"}}
```
Comment (delvh) [inline templates/org/menu.tmpl:21]: What about
```suggestion
				{{svg "octicon-people"}}&nbsp;{{$.locale.Tr "org.members"}}
```
?

File: templates/user/overview/header.tmpl [hunk 2]
```diff
@@ -39,7 +39,7 @@
 			{{if .ContextUser.IsOrganization}}
 				{{if .IsOrganizationMember}}
 					<a class="item" href="{{$.OrgLink}}/members">
-						{{svg "octicon-organization"}}&nbsp;{{$.locale.Tr "org.people"}}
+						{{svg "octicon-organization"}}&nbsp;{{$.locale.Tr "org.members"}}
```
Comment (delvh) [inline templates/user/overview/header.tmpl]: Shouldn't this svg be changed too then?

File: templates/org/menu.tmpl (hunk 1 above)
Comment (yardenshoham) [reply templates/org/menu.tmpl:21]: That's already used for "Teams"

File: templates/user/overview/header.tmpl (hunk 2 above)
Comment (yardenshoham) [reply templates/user/overview/header.tmpl]: Thanks

**Review by delvh — APPROVED** [review]

**Review by lunny — APPROVED** [review]

**Comment by lunny** [comment]
Maybe we should merge the two tabs `members` and `teams` as one?

**Comment by yardenshoham** [comment]
Maybe, but until we decide we can merge this PR

**Pull request merged**